src/redis-cli.c

```diff
@@ -7417,8 +7417,14 @@ static int getDbSize(void) {
 
     reply = redisCommand(context, "DBSIZE");
 
-    if(reply == NULL || reply->type != REDIS_REPLY_INTEGER) {
-        fprintf(stderr, "Couldn't determine DBSIZE!\n");
+    if (reply == NULL) {
+        fprintf(stderr, "\nI/O error\n");
```

Comment:
was there any specific reason for the extra newline?
```suggestion
        fprintf(stderr, "Couldn't determine DBSIZE: I/O error\n");
```

Reply:
There are many occurrences of this in the file, I just want to keep this.

Reply:
maybe in some of them the newline is needed, and in others it doesn't.
it looks very odd that we have 3 failure prints, some have it and some doesn't.
it means that in that specific case either the other two are missing it or this one is excessive.

maybe there's a difference when things are started from command line, like `redis-cli --bigkeys` and things that can be executed from the interactive prompt.

can you please look into it?

Reply:
All `I/O error` are in non interactive mode.  
In fact I think `I/O error` rarely happens. I never saw this output. So leave it as it is is OK.
The failure prints, some places do more strictly and some not. Like `COMMAND`, if it failed, It will not let user known.
Some places check error and return type and some places just check type. The first thinks server is unreliable and the second thinks server is reliable. Maybe just by different people.
IMHO I don't see this difference is  made by purpose.

Reply:
ok.. one small step at a time...